src/components/filterdialog/filterdialog.js

```diff
@@ -234,6 +252,39 @@ class FilterDialog {
         triggerChange(this);
     }
 
+    /**
+         * @private
+         */
+    onResetAllFilters(context) {
+        const query = this.options.query;
+        query.Filters = null;
+        query.IsFavorite = null;
+        query.VideoTypes = null;
+        query.SeriesStatus = null;
+        query.Is4K = null;
+        query.IsHD = null;
+        query.IsSD = null;
+        query.Is3D = null;
+        query.HasSubtitles = null;
+        query.HasTrailer = null;
+        query.HasSpecialFeature = null;
+        query.HasThemeSong = null;
+        query.HasThemeVideo = null;
+        query.IsMissing = null;
+        query.ParentIndexNumber = null;
+        query.Genres = null;
+        query.Tags = null;
+        query.Years = null;
+        query.OfficialRatings = null;
+        query.IsUnaired = null;
+        query.SeriesStatus = null;
+        query.StartIndex = 0;
```

Comment:
These seem to be duplicated in a lot of places. Mainly [here](https://github.com/jellyfin/jellyfin-web/blob/30117bcc860524964ab73ce9507c0e611844bea4/src/scripts/settings/userSettings.js#L23) and [here](https://github.com/jellyfin/jellyfin-web/blob/30117bcc860524964ab73ce9507c0e611844bea4/src/components/filterdialog/filterIndicator.js#L5). Maybe we could move them to a common constants file?

Reply:
@viown thanks for the suggestion  
After searching for a similar scenario in the source i am still not sure where to place this constants file!?  
<details>
  <summary>other constants in a separate file</summary>
 
```sh
$ find src -type f | grep -i constant
src/apps/stable/features/playback/constants/playbackManagerEvent.ts
src/apps/stable/features/playback/constants/mediaSegmentAction.ts
src/apps/stable/features/playback/constants/playerEvent.ts
src/apps/experimental/features/preferences/constants/locales.ts
src/apps/dashboard/features/plugins/constants/categoryLabels.ts
src/apps/dashboard/constants/helpLinks.ts
src/constants/time.ts
```
</details>

what file path would you suggest for such a constants file?

Reply:
I'd say **src/constants** makes sense.

Reply:
@viown done in 2bd24968e3dd47d125de62db8bdc1de5b5c33fd3